test/integration/resource-detail.spec.js

```diff
@@ -0,0 +1,12 @@
+const { resourceDetailPage } = require('./support/integrationSpecHelper');
```

Comment:
Is there another resource we will be displaying ?  e.g. resource-detail vs detail ?